patroni/api.py

```diff
@@ -439,8 +439,8 @@ def get_postgresql_status(self, retry=False):
                     "FROM (SELECT (SELECT rolname FROM pg_authid WHERE oid = usesysid) AS usename,"
                     " application_name, client_addr, w.state, sync_state, sync_priority"
                     " FROM pg_catalog.pg_stat_get_wal_senders() w, pg_catalog.pg_stat_get_activity(pid)"
-                    " INNER JOIN pg_replication_slots AS rs ON rs.active_pid = pid"
-                    " WHERE application_name slot_type = 'physical') AS ri")
+                    " LEFT JOIN pg_replication_slots AS rs ON rs.active_pid = pid"
+                    " WHERE slot_type IS NULL OR slot_type = 'physical') AS ri")
```

Comment:
Responding to:
> Doing JOIN with pg_replication_slots will make the resultset empty even if all replicas are successfully streaming.

We found that this change limits results to the physical slots that we would expect, plus WAL senders that are not associated with a replication slot.

To confirm this in manual testing, we adjusted the join condition to `rs.active_pid = 0`, which still showed all the slots on our host. So, this query should only exclude WAL senders associated with a logical (or non-physical) replication slot.